lib/variable.js

```diff
@@ -49,15 +49,16 @@ module.exports = function getVariableAccessors (container, varName, defValue) {
         value = Buffer.from(value, 'base64').toString()
       }
 
-      const args = [
-        generateRaiseError(value),
-        value
-      ].concat(Array.prototype.slice.call(arguments))
-
-      return accessor.apply(
-        accessor,
-        args
-      )
+      const args = [value].concat(Array.prototype.slice.call(arguments))
+
+      try {
+        return accessor.apply(
+          accessor,
+          args
+        )
+      } catch (error) {
+        generateRaiseError(value)(error.message)
```

Comment:
Could `generateRaiseError` be changed to no longer return a function, so this would become `generateRaiseError(error.message)`?

Reply:
Good shout.  Yup, will change!

Reply:
Good to go!